docsrc/imap/developer/developer-testing.rst

```diff
@@ -60,10 +60,6 @@ Install and configure Cassandane
         sudo cpan -i Mail::JMAPTalk
         sudo cpan -i Math::Int64
 
-   With recent perl versions ``Test::Unit::TestRunner`` cannot be installed with ``cpan``.  Download
```

Comment:
There is missing a `cpan -i` command above, which installs Test::Unit::TestRunner, or is this somehow an indirect, implicit dependency?

Reply:
It's part of `libtest-unit-perl` which is generally expected to have been installed by apt-get (or similar) in the previous step.  If you haven't used a system package for whatever reason, then it's implied that you'd need to install those dependencies by some other means (such as CPAN).  But this list of CPAN modules is the extra stuff that's not available via apt-get, so it's correct that Test::Unit is missing from it.

Reply:
Last time I added cpan-packages in the documentation I was not aware, that first some packages need to be installed over apt-get.  In fact I had no apt-get, so I wrote what I have installed with `cpan -i`.